src/edu/csus/ecs/pc2/shadow/RemoteRunSubmitter.java

```diff
@@ -174,9 +175,10 @@ private Account getSubmitterId(IInternalContest internalContest, String clientId
         // Sort by site, type then id.
         Arrays.sort(accounts, new AccountComparator());
 
-        // serach all site's logins
+        // search all site's logins
         for (Account account : accounts) {
-            if (account.getClientId().getName().equals(clientIdString)) {
+            if (account.getClientId().getName().equals(clientIdString) ||
+                account.getAliasName().equals(clientIdString)) {
```

Comment:
Same comment as above:  I suspect this was intended to call `account.getDisplayName()` rather than `account.getAliasName()`....  yes?

Reply:
The intention was `account.getAliasName()` as discussed in the previous comment.